### PR TITLE
Change Sequencer Cache Logging Level

### PR DESCRIPTION
**Pull request by Maithem**
## Overview
Changed evict logs to trace.

Why should this be merged: Polluting server logs when debug is enabled 

## Checklist (Definition of Done):

- [x] There are no TODOs left in the code
- [x] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [x] Change is covered by automated tests
- [x] Public API has Javadoc

### CODE REVIEW

**Review by zhangn49 — APPROVED** [review]

File: infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java [hunk 1]
```diff
@@ -121,13 +121,18 @@ private int invalidateSmallestTxVersion() {
 
         int numEntries = 0;
         while (firstAddress() == firstEntry.txVersion) {
-            log.debug("evict items " + numEntries + " with address " + firstAddress());
+            if (log.isTraceEnabled()) {
```
Comment (pankti-m) [inline infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java:124]: is it necessary to check if trace is enabled?  The message will not get logged if logging level is > trace

File: infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java [hunk 2]
```diff
@@ -121,13 +121,18 @@ private int invalidateSmallestTxVersion() {
...
+                log.trace("evict items " + numEntries + " with address " + firstAddress());
```
Comment (pankti-m) [inline infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java:125]: can use log.trace(format, args) to avoid concatenation

**Review by pankti-m — APPROVED** [review]

File: infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java (hunk 1 above)
Comment (zhangn49) [reply infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java:124]: @pankti-m 
https://stackoverflow.com/questions/9520093/why-is-it-necessary-to-use-the-istraceenabled-function-in-jboss

File: infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java (hunk 2 above)
Comment (Maithem) [reply infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java]: Done.

**Pull request merged**